tidb-cloud/manage-user-access.md

```diff
@@ -145,21 +137,17 @@ If you are the owner of an organization, you can perform the following steps to
 
 There are four roles in an organization. The permissions of each role are as follows:
 
-- Owner:
-    - Invite members to join the organization and remove members from the the organization
-    - Configure the roles of organization members
-    - Create and rename projects
-    - Invite members to join a project and remove members from a project
-    - Edit time zone
-    - View bills and edit payment information
-- Member:
-    - Can be invited to join a project and obtain project instance management rights
-- Billing Admin:
-    - View bills and edit payment information
-    - Can be invited to join a project and obtain project instance management rights
-- Audit Admin:
-    - View and configure audit logging
-    - Can be invited to join a project and obtain project instance management rights
+|                                                                                       | Owner | Member | Billing Admin | Audit Admin |
+|---------------------------------------------------------------------------------------|-------|--------|---------------|-------------|
+| Invite members to join the organization, and remove members from the organization | ✅     | ❌      | ❌             | ❌           |
+| Set roles for an organization member                                                  | ✅     | ❌      | ❌             | ❌           |
+| Create and rename projects                                                            | ✅     | ❌      | ❌             | ❌           |
+| Invite members to join a project, and remove members from a project                   | ✅     | ❌      | ❌             | ❌           |
+| Edit time zone                                                                        | ✅     | ❌      | ❌             | ❌           |
+| View bills and edit payment information                                               | ✅     | ❌      | ✅             | ❌           |
+| Join a project                                                                        | /     | ✅      | ✅             | ✅           |
```

Comment:
```suggestion
| Join a project                                                                        | ✅    | ✅      | ✅             | ✅           |
```

Reply:
"Join a project" and "Obtain project instance management rights" are not stated for the **owner** role. I'm not sure if they apply to owner.

Reply:
I think it is both yes. Owners can join a project owned by others and obtain project instance management rights as well. It seems that we also need a technical review of this table.